dense_retriever.py

```diff
@@ -135,7 +135,7 @@ def validate(passages: Dict[object, Tuple[str, str]], answers: List[List[str]],
 def load_passages(ctx_file: str) -> Dict[object, Tuple[str, str]]:
     docs = {}
     logger.info('Reading data from: %s', ctx_file)
-    if ctx_file.startswith(".gzip"):
+    if ctx_file.startswith(".gzip") or ctx_file.startswith(".gz"):
```

Comment:
on a second thought, why do we have different compressed extensions? can we use just one?

Reply:
good point.